tests/integration_tests/instances.py

```diff
@@ -216,6 +216,7 @@ def install_deb(self):
             local_path=integration_settings.CLOUD_INIT_SOURCE,
             remote_path=remote_path,
         )
+        assert self.execute("apt-get update -q").ok
```

Comment:
Why this change? This is going to add a significant amount of time to each image launch, so I'd like to be sure we actually need this.

FWIW I don't think we need the following python3-passlib line - that was actually required in https://github.com/canonical/cloud-init/commit/f755bad5d3c683a7102acdf053c4121a5d568be8#diff-a525ccb597ad0f02ccca3a3660450a4ab7c846137bc358a2cc902c4f1b81b7c8, but shouldn't be required any longer since [now it is only required when crypt doesn't exist](https://github.com/canonical/cloud-init/commit/5ed70cc09d1ce068cc920d50f987cba2afffe46a) (Python > 3.12) and in that case it should be included in cloud-init's dependency list in distro packaging.

Reply:
It's just the first launch that creates the custom image, so it shouldn't be much of a problem, but yes, it was because of the passlib dependency.

Reply:
so then we can remove this line. @catmsred , would you be able to do that?

Reply:
Removed the line and rebased.